ios-xcframework/Podfile

```diff
@@ -74,6 +74,12 @@ end
 post_install do |installer|
   react_native_post_install(installer, REACT_NATIVE_PATH)
 
+  # The following workaround is needed to avoid the error `typedef redefinition with different types ('uint8_t' (aka 'unsigned char') vs 'enum clockid_t')`.
+  # This solution is referenced in https://github.com/facebook/react-native/issues/39568#issuecomment-1762890606.
+  # It will be needed until RCT-Folly version is bumped in React Native to version v2022.08.29.00 or above.
+  # Referece: https://github.com/facebook/folly/commit/4a2410fae65afb85e1fec6d922005054b05de59f
+  __apply_Xcode_12_5_M1_post_install_workaround(installer)
```

Comment:
More info can be found in the Gutenberg PR: https://github.com/WordPress/gutenberg/pull/57667